discovery-provider/plugins/verified-uploads/src/handlers/tracks.js

```diff
@@ -19,12 +19,13 @@ export default async ({ slack }, db, { track_id }) => {
     )
     .where("tracks.track_id", "=", trackId)
     .where("users.is_verified", "=", true)
-    .first()
+    .orderBy("tracks.blocknumber", "desc")
+    .limit(2)
```

Comment:
Get two most recent events, current and the one right before. If there's a presence of a previous one then this is not a track creation and thus not an upload.